scripts/launch-s0-instance.sh

```diff
@@ -188,6 +188,9 @@ usage()
     echo "                              NOTE: The group existance and the open ports in the group aren't checked."
     echo "                                    The script may not have permissions and assumes that the group is configured properly."
     echo
+    echo "  -n | --subnet-id          : Optional. A subnet ID to use instead of the default subnet associated with the default VPC. Obligatory if you have no default VPC."
```

Comment:
```suggestion
    echo "  -n | --subnet-id          : Optional. A subnet ID to use instead of the default subnet associated with the default VPC."
```